docs/voice/call-session-notifications.md

```diff
@@ -0,0 +1,176 @@
+# Call Session Notification
+
+A developer can use call session notification to detect all changes in the status of a call effectively. 
+
+## What is Call Session Notification (CSN)?
+
+A Call Session Notification (Telephony Session Notification) is a series of notifications or events a developer can use to track an end-to-end telephony call in RingCentral. The call can be from an external PSTN or any mobile number to a RingCentral extension, or it can be from a RingCentral Digital Line to an external PSTN number, or it can be between two RingCentral extensions. The Call Session Notification provides detailed end-to-end call sessions and party (endpoints/user) level information to enable developers to take specific actions or build custom reports based on that call information. The Call session notification events can be subscribed both at an Account level or at an Extension level.
+
+Account Level Call Session Notification -  Will track and provide notifications for all the extensions within an account.
+
+Extension Level Call Session Notification - Will track calls on a specific extension.
+
+Detailed instructions on the [Notification types](https://developers.ringcentral.com/api-reference/Account-Presence-Event) are available in the API reference.
+
+Detailed instructions on how to subscribe to [RingCentral events](https://developers.ringcentral.com/guide/notifications) or notifications are available in the API reference
+
+
+## What are the use cases?
+
+Call Analytics -  Developers can use the raw data from call session notification events and build call analytics dashboards replicating live reports which use the same event streams. Some of the standard call analytics metrics and their corresponding formulas are:
```

Comment:
Not sure if "live reports" word is still in use. There are multiple Analytics products, like Performance Reports, Historical Reports, etc. Maybe we should name say "replicating RingCentral Analytics reports"?